README.md

```diff
@@ -74,6 +78,11 @@ See [CDP Agentkit Core](./cdp-agentkit-core/README.md) to get started!
 Langchain Toolkit extension of AgentKit. Enables agentic workflows to interact with onchain actions.
 See [CDP Langchain](./cdp-langchain/README.md) to get started!
 
+### farcaster-langchain
+
+Langchain Toolkit extension for Farcaster. Enables agentic workflows to interact with Farcaster, such as to post a tweet.
+See [Farcaster Langchain](./farcaster-langchain/README.md) to get started!
```

Comment:
Can you change this to `./farcaster-langchain/typescript/README.md` and add a quick readme there with getting started info? Could be based off https://github.com/coinbase/agentkit/blob/master/cdp-langchain/typescript/README.md